cmd/keymasterd/app.go

```diff
@@ -601,7 +601,7 @@ func (state *RuntimeState) setNewAuthCookie(w http.ResponseWriter, username stri
 		return "", err
 	}
 	expiration := time.Now().Add(time.Duration(maxAgeSecondsAuthCookie) * time.Second)
-	authCookie := http.Cookie{Name: authCookieName, Value: cookieVal, Expires: expiration, Path: "/", HttpOnly: true, Secure: true}
+	authCookie := http.Cookie{Name: authCookieName, Value: cookieVal, Expires: expiration, Path: "/", HttpOnly: true, Secure: true, SameSite: 4}
```

Comment:
instead of "4", can you please use SameSite.SameSiteStrictMode ? That way we are future proof.

Reply:
Done